apis/filters/parse.go

```diff
@@ -150,3 +157,33 @@ func FromParam(p string) (Args, error) {
 	}
 	return args, nil
 }
+
+//FromFilterOpts parse key=value to Args string from cli opts
+func FromFilterOpts(filter []string) (Args, error) {
+	filterArgs := NewArgs()
+
+	for _, f := range filter {
+		var err error
+		filterArgs, err = ParseFlag(f, filterArgs)
+		if err != nil {
+			return filterArgs, err
+		}
+	}
+	return filterArgs, nil
+}
+
+// Validate compared the set of accepted keys against the keys in the mapping.
+// An error is returned if any mapping keys are not in the accepted set.
+func (args Args) Validate(accepted map[string]bool) error {
+	for name := range args.fields {
+		if !accepted[name] {
+			return errors.New("Invalid filter " + name)
```

Comment:
could we avoid to start with a capital letter in the error string? The reason is here https://github.com/golang/go/wiki/Errors#errors

Reply:
could you add accepted filter name ? make user more clearly

Reply:
> could you add accepted filter name ? make user more clearly

Is there a need to do here?  I find that other validate func in pouch also not do this, maybe We can do this by adding docs or cli usage? wdyt? @Ace-Tang